x-pack/plugins/endpoint/public/applications/endpoint/store/endpoint_list/types.ts

```diff
@@ -47,8 +47,13 @@ export interface EndpointData {
 export interface EndpointListData {
   endpoints: EndpointData[];
   request_page_size: number;
-  request_index: number;
+  request_page_index: number;
   total: number;
 }
 
 export type EndpointListState = EndpointListData;
+
+export interface EndpointListPagination {
+  pageIndex: number;
+  pageSize: number;
+}
```

Comment:
Could we move these types to the `applications/endpoint/types.ts` file? @paul-tavares suggested this approach and the file was added when we merged the basic alert list PR.

Reply:
@peluja1012 - Yes, we'll move it (actually mostly delete this, since it was a copy of the server types as a temporary work-around).
I actually opened a separate issue a few days ago (see elastic/endpoint-app-team/issues/131 ) to handle the refactoring items so that @parkiino could continue to focus on getting the list functionality working.